README.md

```diff
@@ -6,6 +6,9 @@ Portfolio to Microverse course
 > Professional Portfolio using HTML, CSS
 
 ![screenshot](./app_screenshot_1.png)
+![screenshot](./app_screenshot_2.png)
+![screenshot](./app_screenshot_3.png)
+![screenshot](./app_screenshot_4.png)
```

Comment:
[Optional] I will recommend you always put screenshots of just your project display without the development tools view. That way it will look more presentable.